string map_name
geometry_msgs/Pose sensor_pose
geometry_msgs/Point[] points
std_msgs/ColorRGBA[] colors
float32[] weights
int8 action

int8 ACTION_INTEGRATE = 0

---
int32 integrated_points
int8 status 


int8 STATUS_OK=0
int8 STATUS_FAIL=10